cloud/pkg/edgecontroller/manager/common.go

```diff
@@ -8,14 +8,17 @@ import (
 	"github.com/kubeedge/kubeedge/pkg/metaserver/util"
 )
 
+type EventFilterFunc func(oldObj, newObj interface{}) bool
+
 // Manager define the interface of a Manager, configmapManager and podManager implement it
 type Manager interface {
 	Events() chan watch.Event
 }
 
 // CommonResourceEventHandler can be used by configmapManager and podManager
 type CommonResourceEventHandler struct {
-	events chan watch.Event
+	events      chan watch.Event
+	eventFilter EventFilterFunc
```

Comment:
changed to updateFilter? now it's only for update.
And I think whether we can also add `AddFilter` `DeleteFilter`, so that we can also filter some on add or delete, like filter out Not edge node related resources

Reply:
```
// Filter filter the event before enqueuing the key
type EventFilter interface {
	// Create returns true if the Create event should be processed
	Create(obj interface{}) bool

	// Delete returns true if the Delete event should be processed
	Delete(obj interface{}) bool

	// Update returns true if the Update event should be processed
	Update(oldObj, newObj interface{}) bool
}
```

Reply:
done